docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -120,13 +120,16 @@ SQL support
 -----------
 
 The connector provides **read access and limited write access** to data and
-metadata in SQL Server. The following section lists all supported SQL statements
-and provides further details.
+metadata in SQL Server. The following section lists supported SQL statements and
+provides further details.
 
 .. important::
-  No other SQL statements are supported.
 
-Read access:
+    The connector supports the following SQL statements in addition to the
+    :ref:`globally available statements <sql-globally-available>`. No other SQL
+    statements are supported.
+
+Read operations:
```

Comment:
Remove

Reply:
Same question.. does that mean they all work for all connectors.. then we should reword the important box..

Reply:
They work for all connectors

Reply:
okay .. changed accordingly